MIGRATING.md

```diff
@@ -21,9 +21,18 @@ major releases of `cosmwasm`. Note that you can also view the
 
   If you were using cosmwasm-std's `ibc3` feature, you can remove it, as it is
   the default now. Depending on your usage, you might have to enable the
-  `stargate` feature instead, since it was previously implied by `ibc3`. Also
-  remove any uses of the `backtraces` feature. You can use a `RUST_BACKTRACE=1`
-  env variable for this now.
+  `stargate` feature instead, since it was previously implied by `ibc3`.
+
+  Also remove any uses of the `backtraces` feature. You can use a
+  `RUST_BACKTRACE=1` env variable for this now.
+
+  If you were using `cosmwasm-std` with `default-features = false`, you probably
+  want to enable the `std` feature now, as we might move certain existing
+  functionality to that feature in the future to support no_std environments:
+
+  ```toml
```

Comment:
GitHub is complaining about the invalid toml syntax. But I think the highlighting is not so relevant here. I'd just remove the `toml` or fix the syntax.